lib/src/metta/runner/stdlib.rs

```diff
@@ -1012,6 +948,302 @@ impl Grounded for MatchOp {
     }
 }
 
+/// The op atoms that depend on the pkg_mgmt feature
+#[cfg(feature = "pkg_mgmt")]
+pub(crate) mod pkg_mgmt_ops {
+    use super::*;
+
+    //QUESTION: Do we want to factor these catalog management ops and specialized loading
+    // ops into a separate module?  The argument for "yes" is that the it avoids polluting
+    // the namespace with ops that are seldom used.  The argument for "no" is that importing
+    // the module to use the ops is another step users must remember.
```

Comment:
I moved `catalog-list!`, `catalog-update!`, and `catalog-clear!` into a separate module in fa92d15.  I close to keep `register-module!` and `git-module!` in stdlib because they represent different functionality from a user's perspective.

To use the catalog ops, you now need to import them.  Like this:
```
!(import! &self catalog)
!(catalog-list! all)
!(catalog-update! all)
```